projects/kit/directives/highlight/highlight.options.ts

```diff
@@ -0,0 +1,5 @@
+import {tuiCreateOptions} from '@taiga-ui/cdk/utils/di';
+
+export const [TUI_HIGHLIGHT_OPTIONS, tuiHighlightOptionsProvider] = tuiCreateOptions({
+    highlightColor: 'var(--tui-service-selection-background)',
+} as const);
```

Comment:
You need to remove `as const` otherwise it is not `string` but exactly `var(--tui-service-selection-background)` would only be allowed.

Reply:
Sorry, my fault (
[fede712](https://github.com/taiga-family/taiga-ui/pull/8812/commits/fede712345c87ee2ef7f12a2c37275bc66116d55)